export-users.js

```diff
@@ -0,0 +1,49 @@
+require('dotenv').config();
```

Comment:
@SapeleD3 Moses, I'm wondering whether, even though I realize that you are just mapping out options, it would make sense to port this to TS? In my experience, people often start with JS with a goal to switch over eventually, but it doesn't seem to happen. 

Most recently, Thabung did that on the FE. He reasoned that he just wanted to get something working, and that it would be super easy to convert it later to TS. But no time ever seems right, and we are more and more entrenched in JS on our FE now.

Reply:
@YizYah yeah i though so too, will update it to Ts